File: content/code-examples/tested/csharp/driver/TimeSeries/SecondaryIndexes.snippet.sort-with-secondary-index.cs
var matchStage = Builders<BsonDocument>.Filter.Gte("timestamp",
    new DateTime(2022, 1, 15, 0, 0, 0, DateTimeKind.Utc));

var pipeline = new EmptyPipelineDefinition<BsonDocument>()
    .Match(matchStage)
    .Sort(new BsonDocument("timestamp", 1));

var result = _collection?.Aggregate(pipeline).ToList();

File: content/code-examples/tested/python/pymongo/timeseries/secondary_indexes.snippet.sort-with-secondary-index.py
match_stage = {"$match": {"timestamp": {"$gte": datetime(2022, 1, 15, 0, 0, 0, tzinfo=timezone.utc)}}}
sort_stage = {"$sort": {"timestamp": 1}}

pipeline = [match_stage, sort_stage]

result = list(collection.aggregate(pipeline))
